input/pages/artifact-repository-service.md

```diff
@@ -191,7 +191,17 @@ The _review_ operation supports applying a review to an existing artifact, regar
 
 ##### Approve
 
-The _approve_ operation supports applying an approval to an existing artifact, regardless of status. The operation sets the _date_ and _approvalDate_ elements of the approved artifact, and is otherwise only allowed to add _artifactComment_ elements to the artifact, and to add or update an _endorser_.
+The _approve_ operation supports applying an approval to an existing artifact, regardless of status. The operation sets the _date_ and _approvalDate_ elements of the approved artifact, and is otherwise only allowed to add an [_artifactAssessment_](http://hl7.org/fhir/uv/crmi/StructureDefinition/crmi-artifactAssessmentArtifact) resource to the repository.
+
+The following parameters SHOULD be supported for the operation:
+
+* **id**: The server-specific id of the artifact to be approved.
+* **approvalDate**: The date on which the artifact was approved for release. If this parameter is not provided the operation will infer the date to be the current system date on the repository performing the operation.
```

Comment:
Approvals can be done regardless of status, so I don't think they're tied directly to release and I wonder if we should change "The date on which the artifact was approved for release." to something more general like "The date on which the artifact was last approved."